deployment/aws/lambda-functions/create-dbs/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "canton-aws-quickstart-create-dbs",
+  "version": "1.0.0",
+  "description": "AWS lambda function for creating databases for the quickstart deployment",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Canton Team <team@canton.io>",
```

Comment:
Does that email address work? Normally it's canton@digitalasset.com

Reply:
ah right. sorry, was thinking of our twitter handle.